client/my-sites/people/people-list-item/index.tsx

```diff
@@ -179,7 +182,7 @@ const PeopleListItem: React.FC< PeopleListItemProps > = ( {
 	} );
 
 	const tagName = canLinkToProfile() ? 'a' : 'span';
-	const inviteButton = renderInviteButton();
+	const inviteButton = canUserInviteUsers() ? renderInviteButton() : false;
```

Comment:
Looking at the code, it would be nice for me to have a 
```
function shouldShowInviteButton() {
    const canSendInvite = site && userCan( 'promote_users', site );
	return  canReceiveInvite() && ! isInvite && canSendInvite;
}
```

And then use `shouldShowInviteButton` in the return:
```
{ shouldShowInviteButton() && inviteButton }
```